ssc_utils/metric_switcher.py

```diff
@@ -14,95 +14,95 @@ def generate_user_data_cte(self, metric): # <-- whatever you input here in "metr
             String
         """
         # Get the method from 'self'. Default to a lambda.
-        method = getattr(self, metric, lambda: "Invalid metric")
+        metric_clean = metric.replace('--','_').replace('-','_')
+        method = getattr(self, metric_clean, lambda: "Invalid metric")
+        
         # Call the method as we return it
         return method()
     
     def possible_metrics(self):
         # Possible metrics to use for MDE (same as current calculator)
         # may want to make this consistent with the primary metrics available in exp dash in the future
         metrics = [
-            'capped_tvt',
-            'new_viewer_first_day_capped_tvt',
-            'signups',
-            'activations', 
+            'tvt',
+            'tvt-capped',
+            'tvt-capped_new_visitors',
+            'registration-did_signup',
+            'registration-did_activate', 
             'visits',
-            'viewer_conversion',
-            'new_viewer_first_day_conversion',
-            'new_user_1_to_8_days_retained',
-            'all_user_retained_in_experiment_timeframe',
+            'conversion',
+            'conversion--new_visitors',
```

Comment:
Same with these, we have the 5 minute versions in `primary_metrics`. Can you add those and remove these (if you're not using them)?